symlink dirs cli docs (#252)
cylc/cylc-doc: src/user-guide/installing-workflows.rst

Upon running ``cylc install``, symlinks for the directories ``run``, ``log``,
``share``, ``share/cycle`` and ``work`` will be created in accordance with
the symlink rules for ``localhost`` as defined in
:cylc:conf:`global.cylc[install][symlink dirs]`.

> This is overridable via the command line option ``--symlink-dirs="log=$DIR,

Reviewer: ```suggestion
You can override the default locations set in :cylc:conf:`global.cylc` using the command line option ``--symlink-dirs="log=$DIR,
```

run=/path/to/dir,..."``, where the directories supplied will be used to create
symlinks, rather than the ones specified in

> :cylc:conf:`global.cylc[install][symlink dirs]`.

Reviewer: IMO 
- Better to explicitly use a noun rather than "this".
- "overrideable" - possibly not the easiest construction to read. 

```suggestion
You can override the default locations set in :cylc:conf:`global.cylc[install][symlink dirs]` using the command
line option ``--symlink-dirs="log=$DIR, run=/path/to/dir,..."``.
```

.. note::

   If configuring symlink dirs on the command line, the global configured
   symlink dirs will not be used to source directories not included in
   the command line list.

> Use `--symlink-dirs=""` to skip making symlinks.

Reviewer: ```suggestion
To skip making symlinks
"""""""""""""""""""""""

Use `--symlink-dirs=""`
